source/guide_gitea.rst

```diff
@@ -114,7 +120,8 @@ If the verification is fine, we get a ``gpg: Good signature from "Teabot <teabot
 
 .. code-block:: console
 
-  [isabell@stardust ~]$ chmod u+x ~/gitea/gitea
+  [isabell@stardust ~]$ chmod u+x --verbose ~/gitea/gitea
```

Comment:
same here, you can remove the `--verbose` to keept it simple

Reply:
to keep it short!